pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/image-pull-secrets.test.yaml

```diff
@@ -108,19 +108,6 @@ tests:
       authForMain | assertThat(. == "foo:")
       authForCollector | assertThat(. == "foo:")
 
-- name: "default image pull secret are referenced in service accounts"
```

Comment:
I thought I described the reason sufficiently in [this commit's description](https://github.com/stackrox/stackrox/pull/8839/commits/7749887b8ca0a4ef022a09a6b241b132815ccbb7). Perhaps you just missed it, but in case it's not sufficient, or I'm wrong in my assumption somehow, please let me know.